docs/docs/kubernetes/cli/cis.md

```diff
@@ -0,0 +1,161 @@
+| 5.7.1  | Create administrative boundaries between resources using namespaces (Manual)                           | Use namespaces to isolate your Kubernetes objects.                                                                                                                                                                                                             |
+| 5.7.2  | Ensure that the seccomp profile is set to docker/default in your pod definitions                       | Enable docker/default seccomp profile in your pod definitions.                                                                                                                                                                                                 |
+| 5.7.3  | Apply Security Context to Your Pods and Containers                                                     | Apply Security Context to Your Pods and Containers                                                                                                                                                                                                             |
+| 5.7.4  | The default namespace should not be used                                                               | Kubernetes provides a default namespace, where objects are placed if no namespace is specified for them                                                                                                                                                        |
+
+## CLI Commands
+
+Scan a full cluster and generate a complliance CIS-Benchmark summary report:
+
+```
+$ trivy k8s cluster --compliance=cis --report summary
+```
+
+![k8s Summary Report](../../../imgs/trivy-cis-summary.png)
+
+***Note*** : The `Issues` column represent the total number of failed checks for this control.
+
+
+An additional report is supported to get all of the detail the output contains, use `--report all`
+```
+$ trivy k8s cluster --compliance=cis --report all
+```
+Report also supported in json format examples :
+
+```
+$ trivy k8s cluster --compliance=cis --report summary --format json
+```
+
+```
+$ trivy k8s cluster --compliance=cis --report all --format json
+```
+
+## Custom compliance report
+
+The Trivy K8s CLI allows you to create a custom compliance specification and pass it to trivy for generating scan report .
+
+The report is generated based on scanning result mapping between users define controls and trivy checks ID.
+The supported checks are from two types and can be found at [Aqua vulnerability DB](https://avd.aquasec.com/):
+- [misconfiguration](https://avd.aquasec.com/misconfig/)
+- [vulnerabilities](https://avd.aquasec.com/nvd) 
```

Comment:
It is a bit tricky, but a new line is needed for bullet points in mkdocs.

```suggestion
The supported checks are from two types and can be found at [Aqua vulnerability DB](https://avd.aquasec.com/):

- [misconfiguration](https://avd.aquasec.com/misconfig/)
- [vulnerabilities](https://avd.aquasec.com/nvd) 
```

You can build the doc by `make mkdocs-serve`.

![image](https://user-images.githubusercontent.com/2253692/209803927-e63d8b0e-adff-4227-b45a-8ed56567a7c3.png)